.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+    pull_request:
+    push:
+        branches: [ master ]
+
+jobs:
+    run:
+        runs-on: ubuntu-18.04
+        strategy:
+            fail-fast: false
+            matrix:
+                php:
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                minimum_versions: [false]
+                include:
+                    - description: 'Minimum version'
+                      php: '7.2'
+                      minimum_versions: true
+
+        name: PHP ${{ matrix.php }} ${{ matrix.description }}
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - uses: actions/cache@v2
+              with:
+                  path: ~/.composer/cache/files
+                  key: ${{ matrix.php }}-${{ matrix.symfony-versions }}
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+
+            - name: Install dependencies higth
```

Comment:
Typo?

Reply:
I'm sorry, but I didn't understand you a bit here, what do you mean?

Reply:
`high
```suggestion
            - name: Install dependencies
```